modules/codegen/src/test/scala/core/issues/Issue33.scala

```diff
@@ -0,0 +1,95 @@
+package tests.core.issues
+
+import _root_.io.swagger.parser.SwaggerParser
+import cats.instances.all._
+import com.twilio.swagger._
+import com.twilio.guardrail.generators.AkkaHttp
+import com.twilio.guardrail._
+import com.twilio.guardrail.EnumDefinition
+import com.twilio.guardrail.tests._
+import org.scalatest.{FunSuite, Matchers}
+import support.SwaggerSpecRunner
+
+import scala.meta._
+
+class Issue33 extends FunSuite with Matchers with SwaggerSpecRunner {
+  val swagger: String = s"""
+    |swagger: "2.0"
+    |info:
+    |  title: Whatever
+    |  version: 1.0.0
+    |host: localhost:1234
+    |schemes:
+    |  - http
+    |definitions:
+    |  Bar:
+    |    type: string
+    |    enum:
+    |      - v1
+    |      - v2
+    |paths:
+    |  /foo:
+    |    get:
+    |      operationId: getFoo
+    |      parameters:
+    |      - name: CustomHeader
+    |        in: header
+    |        type: string
+    |        x-scala-type: Bar
+    |        required: true
+    |      - name: CustomFormData
```

Comment:
I'd just remove the form parameters entirely, just having a single header parameter. In this example, `Bar` being an enumeration is really good, as this isn't supported by OpenAPI 2.0 currently.